cmd/oras/internal/display/metadata/interface.go

```diff
@@ -28,9 +28,15 @@ type PushHandler interface {
 	OnCompleted(root ocispec.Descriptor) error
 }
 
+// Renderer renders metadata information when an operation is complete.
+type Renderer interface {
+	OnCompleted() error
```

Comment:
```suggestion
	Render() error
```

Reply:
I think `OnCompleted` is a better name. `Render` is confusing and users wouldn't know what it does.

Reply:
Combining with the package name, `metadata.Render` should provide enough information on what was being operated on.